console/console.go

```diff
@@ -260,9 +260,9 @@ func (c *Console) Welcome() {
 			modules = append(modules, fmt.Sprintf("%s:%s", api, version))
 		}
 		sort.Strings(modules)
-		c.jsre.Run("(function () { console.log(' modules: " + strings.Join(modules, " ") + "'); })();")
+		fmt.Fprintln(c.printer, " modules:", strings.Join(modules, " "))
```

Comment:
Any particular reason for converting these to fmt, but not the welcome message part?